linux_os/guide/system/auditing/auditd_configure_rules/audit_login_events/audit_rules_login_events_faillog/tests/correct_rules.pass.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+#
+
```

Comment:
The rule audit_rules_login_events_faillog is only on sle12,ubuntu2004. Do you test it on any of these 2 systems?

Reply:
Hi Jan, I wasn't able to test in these 2 systems but I cant't see a reason to fail there, since the test is only exclusive. I mean that similar tests already passed there. In this one specifically we are only excluding other systems due that specific file (faillog) be typical for these two systems. I will try to prepare a test environment for these systems soon to confirm. But I don't think it is a blockage, considering the test case.